src/core_modules/capture-core/utils/validation/validators/form/expiredPeriod.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+import { getFixedPeriodByDate } from '@dhis2/multi-calendar-dates';
+import { pipe } from 'capture-core-utils';
+import { convertValue as convertClientToServer } from 'capture-core/converters/clientToServer';
+import { convertValue as convertFormToClient } from 'capture-core/converters/formToClient';
```

Comment:
`import { convertClientToServer, convertFormToClient } from 'capture-core/converters` should work after merge of: https://github.com/dhis2/capture-app/pull/4128.

Great if we can change to relative path imports when we are within the same "module" though (the converters and the expiredPeriod file are both inside `capture-core`). This also goes for the other `capture-core` imports in this file. You still need the fix from the other PR.